app/models/round.rb

```diff
@@ -3,4 +3,8 @@ class Round < ApplicationRecord
   belongs_to :team
   belongs_to :movie, optional: true
   has_many :screenings, dependent: :destroy
+
+  def number_in_team
```

Comment:
Just some ideas (not sure if better)

- `index`, without nothing else (the index of the round is by the games rules inside the team, nothing else makes sense neither now or in the future)
- `ordinal`/`ordinal_number`
- or the above with `team_...` before or `...in_team` after.

Reply:
Renamed to index and started using id.